test/queue/QueueTest.java

```diff
@@ -0,0 +1,103 @@
+/**
+ * The queue package contains our Queue, QueueTest, QueueNode and QueueTest 
+ * classes.
+ * 
+ * @author Scott McDonald and Jon Cronin
+ */
+package queue;
+
+import junit.framework.TestCase;
+import static org.junit.Assert.*;
+import org.junit.Test;
+import java.lang.Integer;
+
+/**
+ * Test class used to validate our simple Queue ADT.
+ * @author Scott McDonald and Jon Cronin
+ */
+public class QueueTest extends TestCase {
+    
+    // Default JUnit constructor created by Netbeans. Leaving for now.
+    public QueueTest(String testName) {
+        super(testName);
+    }
+
+    /**
+     * testQueue validates the default Queue constructor.
+     */
+    public void testQueue() {
+        Queue myQ = new Queue();
+        Queue myQSize = new Queue(20);
+        assertTrue(myQ.deque() == null);
```

Comment:
Not a bad idea. I've updated all the asserts i had that made a function call directly in the assert.